pom.xml

```diff
@@ -94,7 +94,7 @@
       <dependency>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>matrix-auth</artifactId>
-        <version>1.2</version>
+        <version>2.0</version>
```

Comment:
Would go with 2.3, it's a much nicer release.

2.4.2 would be a reasonable choice if we increase the core dependency to 2.138.4 (which we should, for https://www.jenkins.io/security/advisory/2018-12-05/#SECURITY-595 and its potential impact on accessible URLs).

Reply:
I just took the oldest "recent enough" to have the failure being solved. Up to the maintainer if they want to use more recent version ;)

Reply:
`matrix-auth` is used only for testing purposes and can be replaced with `MockAuthorizationStrategy`.
I'll do that with updating dependencies for BOM.